src/libraries/System.Reflection.Emit.ILGeneration/tests/ILGenerator/EmitMethodInfo.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class ILGeneratorEmitMethodInfo
+    {
+        public interface IWithIn<T>
+        {
+            void Method(in RuntimeMethodHandle arg);
+        }
+
+        public sealed class WithIn : IWithIn<int>
+        {
+            public void Method(in RuntimeMethodHandle arg)
+            {
+                
+            }
+        }
+
+        [Fact]
+        public void EmitMethodInfo()
+        {
+            var testInstance = new WithIn();
+            var methodType = typeof(IWithIn<int>);
+            var method = methodType.GetMethod("Method");
+
+            ModuleBuilder moduleBuilder = Helpers.DynamicModule(); ;
+            var typeBuilder = moduleBuilder.DefineType("DynamicType", TypeAttributes.Public);
+
```

Comment:
What happens if there are existing calls to `AddInterfaceImplementation` for example?

i.e. is this a breaking change?